tuf/data/types.go

```diff
@@ -17,6 +18,53 @@ import (
 	"github.com/docker/notary"
 )
 
+// GUN type for specifying gun
+type GUN string
+
+func (g GUN) String() string {
+	return string(g)
+}
+
+// NewGUN generates a GUN object from the provided string
+func NewGUN(g string) GUN {
```

Comment:
I will change to GUN("gun"), I was just trying to keep with convention. Will make the change thanks for the feedback :+1: